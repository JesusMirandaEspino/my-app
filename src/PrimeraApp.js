import React from 'react';
import PropTypes from 'prop-types';

const PrimeraApp = ( { saludo }  ) => {

   // const saludo = 'Hola mundo desde react';

    return (
        <section className="fila">  
            <div className="contenedor1">
                <div className="col-full-12">
                    <h1> { saludo } </h1>
                </div>
            </div>
        </section>
    );
    
}

PrimeraApp.propTypes = { 
    saludo: PropTypes.string
}

export default PrimeraApp;