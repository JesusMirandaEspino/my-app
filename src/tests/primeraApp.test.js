import React from 'react';
import '@testing-library/jest-dom';
//  Render import { render } from '@testing-library/react'

import { shallow } from 'enzyme'
import  PrimeraApp  from '../PrimeraApp';




describe( 'Primera App', () => { 


    /*  No se utlizara esta prueba 
    test( ' Debe de mostrar el mensa "Hola soy Goku"',  () => {
        const saludo = 'Hola, soy Goku';
        const { getByText } = render( <PrimeraApp saludo={saludo} />);
        expect( getByText(saludo) ).toBeInTheDocument(  );
    });  */


    test( 'Debe de mostar <PrimeraApp/> correctamente', () => {

        const saludo = 'Hola a todos';
        const wrapper = shallow( <PrimeraApp saludo={ saludo } />);

        expect( wrapper ).toMatchSnapshot(  );

    });




} );
