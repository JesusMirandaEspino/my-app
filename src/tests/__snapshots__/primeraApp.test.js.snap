// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primera App Debe de mostar <PrimeraApp/> correctamente 1`] = `ShallowWrapper {}`;
